.nvmrc

```diff
@@ -0,0 +1 @@
+12.16.1
```

Comment:
This specifies the version of [Node](https://nodejs.org/en/), which is a JavaScript runtime.